op-batcher/compressor/shadow_compressor_test.go

```diff
@@ -110,3 +110,28 @@ func TestShadowCompressor(t *testing.T) {
 		})
 	}
 }
+
+// TestBoundInaccruateForLargeRandomData documents where our bounding heuristic starts to fail
+// (writing at least 128k of random data)
+func TestBoundInaccurateForLargeRandomData(t *testing.T) {
+	var sizeLimit int = 1 << 17
+
+	sc, err := NewShadowCompressor(Config{
+		TargetFrameSize: uint64(sizeLimit + 100),
+		TargetNumFrames: 1,
+	})
+	require.NoError(t, err)
+
+	_, err = sc.Write(randomBytes(t, sizeLimit+1))
```

Comment:
We've recently been moving away from using `crypto/rand` randomness in tests and instead use `math/rand` to make them more deterministic.
I can see that it is practically impossible for this test to fail here given we produce `1 << 17` bytes of randomness, but wondering if we want to take the opportunity to change the randomness in the shadow compressor tests to using a `math/rand.Rand` that we pass to `randomBytes` and that we initialize with a fixed seed in each test.

Reply:
changed to math/rand

Reply:
..and with a deterministic generator.